app/controllers/bus_prediction_controller.rb

```diff
@@ -0,0 +1,27 @@
+class BusPredictionController < ApplicationController
+  def index
+
+  end
+
+  def bus_prediction_from_postcode
+    postcode = params[:postcode]
+    # Rails.logger.debug "Postcode: #{postcode}"
+
+    address = Address.new(postcode)
+    stop_id_list = address.get_nearest_stops
+    # Rails.logger.debug "Stop ID List: #{stop_id_list.inspect}"
+
+    @results = stop_id_list.map do |stop_id|
+      stop_instance = BusStop.new(stop_id)
+      stop_instance.bus_prediction(1)
+    end
+
+    # logger.debug "Postcode: #{postcode}"
+    # logger.debug "Stop ID List: #{stop_id_list.inspect}"
+    # logger.debug "Predictions: #{@results.inspect}"
+    #
+    # Rails.logger.debug "Predictions: #{@results.inspect}"
```

Comment:
In general you should avoid committing commented-out code, especially if it was for debugging.